code/mob/dead/mentor_mouse_observer.dm

```diff
@@ -4,11 +4,14 @@
 	is_respawnable = FALSE
 	locked = TRUE
 
+	start_listen_inputs = list(LISTEN_INPUT_EARS, LISTEN_INPUT_RADIO_GLOBAL, LISTEN_INPUT_MENTOR_MOUSE)
+	default_speech_output_channel = SPEECH_OUTPUT_MENTOR_MOUSE
+	start_speech_outputs = list(SPEECH_OUTPUT_MENTOR_MOUSE)
 	start_listen_languages = list(LANGUAGE_ALL)
 
 	var/image/ping
 	var/ping_id
-	var/mob/the_guy
+	var/mob/mentee
```

Comment:
I changed "the_guy" to "mentee" for accuracy, but lmk if i should revert that and just pr it directly to main or not at all